### PR TITLE
Completed react-form-controls.

### PR DESCRIPTION
**Pull request by stefancarrera**
react form

### CODE REVIEW

File: react-form-controls/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  handleSubmit() {
+    event.preventDefault();
+    console.log('Submitted Email:', this.state.value);
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <label>
```
Comment (Zhiwutian) [inline react-form-controls/src/index.jsx:25]: make sure to add the proper attribute to the label!

File: react-form-controls/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          Email:
+          <input type="text" value={this.state.value} onChange={this.handleChange}></input>
```
Comment (Zhiwutian) [inline react-form-controls/src/index.jsx:27]: make sure to add the `id` and `name` attributes

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Nice work sir, a couple small updates

**Comment by stefancarrera** [comment]
forms

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**